src/components/views/messages/MessageActionBar.tsx

```diff
@@ -442,7 +445,9 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
 
     public render(): JSX.Element {
         const toolbarOpts = [];
-        if (canEditContent(this.props.mxEvent)) {
+        const isFavouritePageCheck = window.location.href.split("/").pop() === "favourite_messages";
```

Comment:
we shouldn't be making assumptions about the URL structure, or whether it's at the root level. We should be able to ask MatrixChat or similar if we're on the page, though.